interface/patient_file/summary/demographics.php

```diff
@@ -723,17 +723,17 @@ function setMyPatient() {
 <?php endif;
         //Patient Portal
 
-        // If patient is deceased, then show this (along with the number of days patient has been deceased for)
+        // If patient is deceased, then show this (along with the deceased date if greater than 90 days)
         $days_deceased = is_patient_deceased($pid);
 if ($days_deceased != null) : ?>
```

Comment:
This could be problematic now that you are returning an array. Would instead simplify above line with:
```
if ($days_deceased) : ?>
```